variables/Variables-home.md

```diff
@@ -0,0 +1,1122 @@
+Variables can be used to change almost everything about how the game works. Currently there is almost no documentation on the website, so as of now this page is just a really long list of variables and descriptions copied and pasted from [here](https://github.com/redeclipse/base/blob/master/config/usage.cfg) and then formatted for the website. However, we are working on sorting these into a series of pages. You can help too by clicking the "edit on GitHub" button on the top right.
```

Comment:
I would also leave it at the link, the link, as the this list will be outdated pretty fast.
Probably explain some commands here in more in depth as it is in the usage.cfg.

Reply:
ok, I can change the link to be the URL, however, I hardly know any commands/variables so that's why this is just a copy and paste. something is better than nothing